src/components/playback/mediasession.js

```diff
@@ -118,19 +127,9 @@ define(['playbackManager', 'nowPlayingHelper', 'events', 'connectionManager'], f
         }
 
         var playState = state.PlayState || {};
-
         var parts = nowPlayingHelper.getNowPlayingNames(item);
-
-        var artist = parts.length === 1 ? '' : parts[0].text;
-        var title = parts[parts.length - 1].text;
-
-        // Switch these two around for video
-        if (isVideo && parts.length > 1) {
```

Comment:
do you have any clue on why was this ever needed?..

Reply:
No idea. It was right, however, given how the variables were first declared, don't know why they didn't do the same for music 😆.